### ISSUE TITLE
Fix orientation

### ISSUE THREAD

**Issue by vaclav-dvorak** [sunshineplan/imgconv#30]
Hello

I was wondering if it would be possible to use imaging Decode function instead of image one and make use of its ability to fix exif orientation.

From their code: https://github.com/disintegration/imaging/blob/master/io.go#L54

As this module is already using it, it seems beneficial to make use of it if possible.

Regards

**Comment by sunshineplan** [comment]
Hello vaclav-dvorak,

To do this, I need to decide whether to add new functions or to add an additional enable auto orientation parameter to the Decode and Open functions.

I tend to add new functions, but this is basically the same as using the imaging package directly.

Any suggestions?

**Comment by vaclav-dvorak** [comment]
Hi

i have no strong opinion on this tbh :( I do not see why the orientation is even turned off in imaging by default... maybe it can screw the image if exif is incorrect... not sure.

What i ended up doing was that instead of

```img, _ := imgconv.Decode(r)`~~

ive used:

```img, _ := imaging.Decode(r, imaging.AutoOrientation(true))```

and it fixed our issue with have with wrongly rotated images from phones.

I think it would be nice to have this options somehwhere in decodeOptions probably.

**Comment by sunshineplan** [comment]
Do you encounter this issue only in converter?
I can add an AutoOrientation option to fix it.
But in imgconv package, I will not do anything, because I would recommend using `imaging.Decode` directly.

**Comment by vaclav-dvorak** [comment]
That is fine i believe.
Problem was that we had some images which were created on phone and had orientation in exif. When we run them through `imgconv.Decode` we got incorrectly rotated images. Running it through `imaging.Decode` we got the correct output.

My proposal was to just have it in imgconv as it would made things easier.
Feel fee to to close this issue as there is really simple workaround anyway.
